ui/components/ItemDetail/TrainingDetail.js

```
@@ -82,7 +79,7 @@ const TrainingDetail = ({ training, isCfa }) => {
 
           try {
             v.prdvUrl = url;
-            dispatch(setTrainingsAndSelectedItem(updatedTrainings, v));
+            setTrainingsAndSelectedItem(updatedTrainings, v);
           } catch (err) {}
```

Having a try/catch without any code into the catch block isn't a good pratice. it creates silent errors.